.config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
i'd add some prefix to the config file so it is more obvious. like config can configure anything

Reply:
i'd add prefix to the file name, not to the JSON. database configuration does not have to do anything with the express so just remove that please